core/src/main/java/org/testcontainers/containers/PortForwardingContainer.java

```diff
@@ -60,6 +60,13 @@ public void exposeHostPort(int port) {
             getSshConnection().requestRemotePortForwarding("", port, "localhost", port);
         }
     }
+    
+    @SneakyThrows
+    public void exposeHostPort(int hostPort, int containerPort) {
```

Comment:
`exposeHostPort(int port)` should delegate to this method now

Reply:
done